### ISSUE TITLE
How might i create a "virtual indoor tour" using Meshroom?

### ISSUE THREAD

**Issue by fatbringer** [alicevision/Meshroom#2434]
Hello! I have previously used Meshroom with recreating 3d models from photographs taken of an object. The photographs were taken similar to using a photo studio with a roundtable. I was using the default photogrammetry pipeline while making very limited changes to the settings in the nodes

However now, i am interested in re-creating a 3D environment of an indoor location, such that i can use it for a virtual indoors tour. 

In my mind, this is conceptually the opposite of what i had done previously i.e. now i am inside the object rather than outside.

How might i achieve this goal?

**Comment by fabiencastan** [comment]
To maximize parallax in your indoor photogrammetry, position yourself along the walls and keep your back against the wall while shooting.
Also take photos at different heights, not just at eye level.

**Comment by fatbringer** [comment]
@fabiencastan alright thanks I'll take note to do it along the walls

1) should I still be using the default photogrammetry pipeline? Any changes I should make?

2) can I use videos instead?

**Comment by fabiencastan** [comment]
1. Yes same pipeline 
2. You can use videos but you will have better results with sharp photos. The main problem with videos is that you can't avoid motion blur.

**Comment by fatbringer** [comment]
@fabiencastan oh yes indeed the frames extracted from the videos have alot of blur....

By any chance, would the process work on 360° photos? Eg those taken by cameras such as an Insta360. 

Though I understand photos with distortions such as fish-eye effect already pose a problem.

**Comment by fabiencastan** [comment]
Yes, you can use 360 images with the [Split360Images node](https://github.com/alicevision/Meshroom/pull/1939).
But the distortion introduces significant variations in resolution, resulting in a decrease in overall quality.
